_docs/faqs.md

```diff
@@ -37,4 +37,4 @@ The National Data Buoy Center harvests new profile observations from the [DAC's
 
 <!-- 2. Can I submit raw glider files to the DAC? -->
 
-<!-- At the moment no, you must convert the raw files to netCDF following the DAC [metadata conventions](/ioosngdac/ngdac-netcdf-file-format-version-2.html). But we're in the process of developing a raw data upload tool that will enable users to  -->
+<!-- At the moment no, you must convert the raw files to netCDF following the DAC [metadata conventions](/glider-dac/ngdac-netcdf-file-format-version-2.html). But we're in the process of developing a raw data upload tool that will enable users to  -->
```

Comment:
Everything here is commented out, no text is created for the page, should we clean this up? (If yes, let's do in another PR to avoid cluttering this one.)

Reply:
I'd leave commented out, but it can't hurt to move forward with that link repair.